consensus/hotstuff/votecollector/factory.go

```diff
@@ -63,7 +63,7 @@ func (f *VoteProcessorFactory) Create(log zerolog.Logger, proposal *model.Propos
 // NewStakingVoteProcessorFactory implements hotstuff.VoteProcessorFactory for
 // members of a collector cluster. For their cluster-local hotstuff, collectors
 // only sign with their staking key.
-func NewStakingVoteProcessorFactory(committee hotstuff.Committee, onQCCreated hotstuff.OnQCCreated) *VoteProcessorFactory {
+func NewStakingVoteProcessorFactory(committee hotstuff.Replicas, onQCCreated hotstuff.OnQCCreated) *VoteProcessorFactory {
```

Comment:
same argument as for [combined_vote_processor_v2.go](https://github.com/onflow/flow-go/blob/4de4231e43a65fe672a54aab4f84971ecac03db6/consensus/hotstuff/votecollector/combined_vote_processor_v2.go), I think we should use the `DynamicCommittee` for all factories here to reject votes from ejected replicas.